components/content/src/taxonomies.rs

```diff
@@ -204,8 +203,8 @@ impl Taxonomy {
         context.insert("terms", &terms);
         context.insert("lang", &self.lang);
         context.insert("taxonomy", &self.kind);
-        context.insert("current_url", &config.make_permalink(&self.kind.name));
-        context.insert("current_path", &format!("/{}/", self.kind.name));
+        context.insert("current_url", &self.permalink);
+        context.insert("current_path", &self.path);
```

Comment:
so we already had them eh

Reply:
Yes, we indeed had both `permalink` and `path` already computed in the `new` function, but `path` was discarded and `permalink` wasn't used. I've made it to match other similar structs e.g., `Path`